Homework1/ParMatrixMult/ParMatrixMult.Tests/MultiplyTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// <copyright file="MultiplyTests.cs" company="_">
+// Marina Popova, 2025, under MIT License.
+// </copyright>
+
+namespace ParMatrixMult.Tests;
+
+public class MultiplyTests
+{
+    private static Matrix firstMatrix;
+    private static Matrix secondMatrix;
+    private static Multiplication multiply = new Multiplication();
```

Comment:
```suggestion
    private static Multiplication multiply = new();
```
Два раза один тип вроде вообще никогда уже писать не надо